src/docs/app-monitoring/user-defined-monitoring.md

```diff
@@ -212,9 +212,43 @@ For detail steps, please read the documents below:
 
 ## Monitor user-defined projects using Alertmanager
```

Comment:
Can we move this section before **Sysdig Monitor** ? To keep the on-cluster prometheus stuff and Sysdig stuff separate.

Reply:
agree! if a team just need alerts but not a full dashboard, this is sufficient.

Reply:
moved the section before sysdig section